build/bundle.css

```diff
@@ -568,13 +569,11 @@ dd .maths {
 
 
 
-
-
 .Q-circuit,
 .Q-circuit-palette {
 
 	position: relative;
-	width: 50%;
+	width: 100%;
```

Comment:
What does this change do ?

Reply:
width: 100% for Q-circuit-palette was the original CSS in Q.js. 50% creates _too small_ a Q-circuit-palette for the browser.